### PR TITLE
Added clearing whitespace when converting header string to array

### PR DESCRIPTION
**Pull request by kadarsys**
Hello. Faced a bug when passing multiple headers in an OPTIONS request. The bug was found only in IE 11. I fixed it. Please see PR

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -2,4 +2,5 @@
 /composer.lock
 /.phpunit.result.cache
 /docs/_site
-/docs/.jekyll-cache
\ No newline at end of file
+/docs/.jekyll-cache
+/phpunit.xml
```
Comment (cspray) [inline .gitignore:6]: I don't believe the phpunit file should be added to ignore.

File: composer.json [hunk 2]
```diff
@@ -6,6 +6,7 @@
     "test": "vendor/bin/phpunit"
   },
   "require": {
+    "php": "^7.0",
```
Comment (cspray) [inline composer.json:9]: I don't want to exclude PHP8 installs. Please update to "^7.0 || ^8.0"

**Review by cspray — CHANGES_REQUESTED** [review]
Hi! Thank you for submitting this fix. I made some comments on minor adjustments that should be made. Once the changes have been put in place I'll be happy to merge this in.

File: .gitignore (hunk 1 above)
Comment (kadarsys) [reply .gitignore:6]: Oк. But I would think that if the git index contains phpunit.xml.dist, then it should be excluded from the index.  This is not true?

Comment (cspray) [reply .gitignore:6]: That's a fair point. The `.dist` is actually a legacy artifact from how I was setting up most of my projects. I would actually prefer to see the `phpunit.xml.dist` file renamed to `phpunit.xml`. We can then remove this from `.gitignore`. No other changes should be necessary.

Comment (kadarsys) [reply .gitignore:6]: Ok

**Review by cspray — APPROVED** [review]

**Comment by cspray** [comment]
Thank you for submitting your PR @kadarsys! I want to make some changes to my CI setup to properly test 7.x and 8.x versions. After that I will cut a new bug fix release with your additions.

**Pull request merged**